addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -40,19 +40,22 @@
                  <span t-esc='props.formatCurrency(props.data.amount_total + props.data.rounding_applied)' class="pos-receipt-right-align"/>
               </div>
             </t>
-            <br/><br/>
 
             <!-- Payment Lines -->
 
-            <div class="paymentlines" t-foreach="props.data.paymentlines" t-as="line" t-key="line_index">
-                <t t-esc="line.name" />
-                <span t-esc="props.formatCurrency(line.amount, false)" class="pos-receipt-right-align"/>
+            <div class="pos-receipt-pay">
```

Comment:
it's nicer to use bootstrap utility classes rather than declaring your own css classes. The pos was recently moved to bootstrap so that's why there are still many custom classes, but it's something that we tend to stay away from. We generally choose to use custom class names when we want to be able to refer to those elements. ( for example in tours or in xpaths )

Also, if you can, it's nicer to try to make the html "compact", so if you can get away with a flatter div structure it's preferable.